AssetManagementTestProject/TestCases/US307_EditUserTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using AssetManagementTestProject.DAO;
+using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestData;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace AssetManagementTestProject.TestCases;
+
+[TestFixture]
+public class US307_EditUserTest : NUnitWebTestSetup
+{    
+    protected ManageUserPage? ManageUserPage; 
+    protected EditUserInfoPage? EditUserInfoPage;
+    protected DetailedUserInfoPage? DetailedUserInfoPage;
+    protected ViewUserDAO.ViewDetailedUser? DetailedUserInfoFromUI;
+    protected GetUserDAO.GetUserResponse? NewUserAfterEdit;
```

Comment:
Can you move them into the parent class?

Reply:
Done